README.md

```diff
@@ -36,7 +36,7 @@ It applies various transformations according to program options before passing t
 | --css-no-default             | HTML      | Do not use default transformation CSS                        |
 | --suppress-nbsp-num          | HTML      | Suppress non-breakable spaces (U+00A0) between numbers       |
 | --ignore-0-space             | HTML      | Suppress zero width space (U+200B)                           |
-| --css-greek-title-plus       | HTML      | Use Greek transliteration from title attribute               |
+| --css-greek-title-plus       | HTML      | By default, if there is some Greek and the text version has the transliteration only (i.e. it's in latin1), and if the HTML also has the transliteration in the title attribute, then that text is applied |
```

Comment:
This is a rather unwieldy description for this table. Perhaps shorten it to `Special handling for Greek (see below)` and then add a brief subsection:
```
### `--css-greek-title-plus`
By default, if there is some Greek and the text version has the transliteration only (i.e. it's in latin1), and if the HTML also has the transliteration in the title attribute, then that text is applied.
```
That gives you space to be clearer about what this is doing (because I don't understand based on the current description).

Reply:
Shortened it to what is on the web site. My preference was to delete it as obviously obsolete, but certain people were opposed, and I guess it doesn't hurt. But whoever wanted it before should already understand it.